src/main/java/org/jahia/modules/reports/bean/ReportLiveContents.java

```diff
@@ -84,31 +86,41 @@ public ReportLiveContents(JCRSiteNode siteNode, String searchPath) {
     @Override public void execute(JCRSessionWrapper session, int offset, int limit)
             throws RepositoryException, JSONException, JahiaException {
         logger.debug("Building jcr sql query");
+        LocalDateTime now = LocalDateTime.now();
```

Comment:
We should probably make it explicit that which locale we use to avoid confusion. So something like this 'now.atOffset(ZoneOffset.systemDefault())' probably.

Reply:
sure